ui/panels-plugin/src/plugins/time-series-chart/VisualOptionsEditor.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Slider, Stack, Typography } from '@mui/material';
+import { VisualOptions, DEFAULT_LINE_WIDTH, DEFAULT_POINT_RADIUS } from './time-series-chart-model';
+
+export interface VisualOptionsEditorProps {
+  value: VisualOptions;
+  onChange: (visual: VisualOptions) => void;
+}
+
+export function VisualOptionsEditor({ value, onChange }: VisualOptionsEditorProps) {
+  const handlePointRadiusChange = (_: unknown, sliderValue: number | number[]) => {
+    const newValue = Array.isArray(sliderValue) ? sliderValue[0] : sliderValue;
```

Comment:
i think it should always return `number` since `number[]` is for range slider?

Reply:
So I fought with this for a bit and I might just be missing something, but when I remove number[] it currently throws:

![image](https://user-images.githubusercontent.com/9369625/202022628-6b45da2a-8336-40ca-b247-d7fe3bef9d4e.png)